tests/test_fetcher_ng.py

```diff
@@ -54,7 +54,7 @@ def setUp(self):
             self.file_contents = target_fileobj.read()
             self.file_length = len(self.file_contents)
 
-        self.rel_target_filepath = os.path.basename(target_filepath)
+        self.rel_target_filepath = os.path.basename(target_filepath).replace("\\","/")
```

Comment:
I think `os.path.basename `always returns the "basename" and does not contain separators. I also didn't see any errors in the CI run related to `test_fetcher_ng.py`. Is this replace here "just in case" or I am missing something?

Reply:
good catch, looks like the correct analysis to me: replace not necessary here

Reply:
The test was failing and this is why I added replace here, but I'll try once again without to double verify.